toolkit/partner-chains-cli/src/setup_main_chain_state/tests.rs

```diff
@@ -90,12 +100,19 @@ fn ariadne_parameters_are_on_main_chain_no_updates() {
 
 #[test]
 fn ariadne_parameters_are_on_main_chain_do_update() {
-	let offchain_mock = OffchainMock::new().with_upsert_d_param(
-		UtxoId::default(),
-		new_d_parameter(),
-		payment_signing_key(),
-		Ok(Some(McTxHash::default())),
-	);
+	let offchain_mock = OffchainMock::new()
+		.with_upsert_d_param(
+			UtxoId::default(),
+			new_d_parameter(),
+			payment_signing_key(),
+			Ok(Some(McTxHash::default())),
+		)
+		.with_upsert_permissioned_candidates(
+			genesis_utxo(),
+			&initial_permissioned_candidates(),
+			payment_signing_key(),
+			Ok(Some(McTxHash::default())),
```

Comment:
Perhaps it is logged only, but please use different values in tests. I mean upsert_d_param cannot return the same as upsert_permissioned_candidates

Reply:
changed